### PR TITLE
Updated Convolutional Neural Network

### PR DESCRIPTION
**Pull request by Dynamo13**
Used keras.layers to add the hidden layers (Maxpooling2D and Convolutional layers) and flatten layer followed by dense layers to give the output .Finally adam optimiser was used with accuracy as metrics to evalute the model

### CODE REVIEW

**Comment by karlnapf** [comment]
nope
